sbncode/EventGenerator/MeVPrtl/Tools/IMeVPrtlDecay.h

```diff
@@ -46,7 +46,9 @@ class IMeVPrtlDecay: virtual public IMeVPrtlStage
 protected:
     double TimeOfFlight(const MeVPrtlFlux &flux, TVector3 decay) {
       // TODO: should the neutrino TOF be subtracted here to get the correct T0?
-      return flux.pos.T() + (flux.pos.Vect() - decay).Mag() * (1. / flux.mom.Beta()) / Constants::Instance().c_cm_per_ns;
+      //
+      // TODO: implement time of flight
```

Comment:
Is this undoing a TOF correction? I think you're the expert here so I defer to you, just wondered about this choice.

Reply:
It is. I had some code to try to implement the TOF of the (massive) LDM. But I decided I didn't trust it so for this production I turned it off. Planning to implement it for the futurue.